Makefile

```diff
@@ -457,40 +431,10 @@ clean:
 
 # swagger
 
-$(GOPATH)/bin/swagger:
-	$(call backup_go_mod)
-	go get github.com/go-swagger/go-swagger/cmd/swagger@v0.23.0
-	$(call restore_go_mod)
-
 .PHONY: swagger
-swagger: api/openapi-spec/swagger.json
-
-pkg/apis/workflow/v1alpha1/openapi_generated.go:
```

Comment:
There's a bug here, the file will never get updated since it's already existing.